GVFS/GVFS.Common/NuGetUpgrader/NuGetUpgrader.cs

```diff
@@ -460,6 +482,7 @@ public NuGetUpgraderConfig(ITracer tracer, LocalGVFSConfig localGVFSConfig)
             public string FeedUrl { get; private set; }
             public string PackageFeedName { get; private set; }
             public string FeedUrlForCredentials { get; private set; }
+            public string CertificateFingerprint { get; private set; }
```

Comment:
Is this only for testing\debugging?  If so, will it be removed before merging to master? (on the surface it seems like a security risk to leave it in the production code)

Reply:
I do plan to remove it before merging. I don't think it actually increases our risk very much by itself, as it:

1) Must be configured in a file that requires admin privileges to access
2) The cert must be trusted by the machine, so it is not sufficient to just set this fingerprint.

However, we can consider this after the initial work here. This is contained in its own commit, so it should be easy to pull out.